src/toxcore/dht/server/mod.rs

```diff
@@ -207,7 +207,6 @@ impl Server {
     /// Enable/disable IPv6 mode of DHT server.
     pub fn enable_ipv6_mode(&mut self, enable: bool) {
         self.is_ipv6_enabled = enable;
```

Comment:
Do we still need it?

Reply:
It's used to filter out received nodes with IPv6 address if we don't support it: https://github.com/tox-rs/tox/blob/f5826d7f88bde1860100895a16cffa424297cc33/src/toxcore/dht/server/mod.rs#L834-L836
Otherwise we'd add it to the `nodes_to_bootstrap` list and then try to ping without success. So it'd only take up a place in `nodes_to_bootstrap` with probably other good nodes eviction.